frontend/src/general/TabPanelH.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Box } from "@material-ui/core";
+
+export default function TabPanelH(props) {
```

Comment:
took a second to understand what the H was - do you think it's worth it to rename this to `HorizontalTabs` to match `VerticalTabs`? or maybe rename `VerticalTabs` so we have consistent naming for our tab files?
actually the way verticaltabs works is different from this file so maybe verticaltabs warrants a refactor issue in the future 🤔 

**tldr** maybe write out the `Horizontal` instead of just `H` since you're importing it as `TabPanel` regardless

Reply:
I think vertical tabs should be renamed, since it's not the tabs themselves but the corresponding panels

but yeah TabPanelH renamed in 6a1b1a7a